dockerfile/cuda11.1.1.dockerfile

```diff
@@ -22,6 +22,7 @@ RUN apt-get update && \
     apt-get install -y --no-install-recommends \
     autoconf \
     automake \
+    bc \
```

Comment:
What change for this?

Reply:
to compare the cuda version in makefile, >=11.8 and <11.8 have different build logic